src/Mado/QueryBundle/Queries/QueryBuilderFactory.php

```diff
@@ -90,12 +98,12 @@ public function setSorting(array $sorting = [])
 
     public function getFilters()
```

Comment:
maybe is better to call this function getAndFilters, because we have also getOrFilters

Reply:
good catch!!